docs/interceptors.md

```diff
@@ -187,6 +187,43 @@ For reference, below is an example legacy GitHub `Interceptor` definition:
 
 For more information, see our [example](../examples/v1beta1/github) of using this `Interceptor`.
 
+#### Adding Changed Files
+
+The GitHub `Interceptor` also has the ability to add a comma delimited list of all files that have changed (added, modified or deleted) for the `push` and `pull_request` events. The list of changed files are added to the `changed_files` property of the event payload in the top-level `extensions` field
```

Comment:
Thanks for adding the docs...it seems like add changed files also supports private github repositories using a personal access token - could we document the fields that are needed to support that here as well?

Reply:
@dibyom Added additional documentation on personal access token